internal/server/backup_handlers.go

```diff
@@ -109,3 +112,44 @@ func parseTimestamp(value string, defaultValue int64) (int64, error) {
 	}
 	return strconv.ParseInt(value, 10, 64)
 }
+
+// @Summary  Schedule a full backup once per routine name.
+// @ID       scheduleFullBackup
+// @Tags     Backup
+// @Param    name query string true "Backup routine name"
+// @Param    delay query int false "Delay interval in milliseconds"
+// @Router   /backup/schedule [post]
+// @Success  202
+// @Failure  404 {string} string ""
+func (ws *HTTPServer) scheduleFullBackup(w http.ResponseWriter, r *http.Request) {
+	routineName := r.URL.Query().Get("name")
+	delayParameter := r.URL.Query().Get("delay")
+	var delayMillis int
+	if delayParameter != "" {
+		var err error
+		delayMillis, err = strconv.Atoi(delayParameter)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+	}
+	if delayMillis < 0 {
+		http.Error(w, "nonpositive delay query parameter", http.StatusBadRequest)
+		return
+	}
+	job := service.GetFullBackupJobByRoutineName(routineName)
```

Comment:
I think, we should move all the actual business logic to scheduler (or even new file in it's package) and leave in handler only parameter validation and execution of those extracted code